lib/pangea/extensions/pangea_room_extension.dart

```diff
@@ -808,6 +812,85 @@ extension PangeaRoom on Room {
     );
   }
 
+  Future<void> archive() async {
+    final participants = await requestParticipants();
+    final students = participants
```

Comment:
Based on testing - it's possible this list could contain the logged in user's ID. Can you filter that out?